[2023-06-15T06:09:35.380+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T06:04:27.115299+00:00 [queued]>
[2023-06-15T06:09:35.385+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T06:04:27.115299+00:00 [queued]>
[2023-06-15T06:09:35.386+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 4
[2023-06-15T06:09:35.394+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): pyspark_train_classifier_model> on 2023-06-15 06:04:27.115299+00:00
[2023-06-15T06:09:35.398+0000] {standard_task_runner.py:57} INFO - Started process 1470 to run task
[2023-06-15T06:09:35.402+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'deploy_prediction_model', 'pyspark_train_classifier_model', 'manual__2023-06-15T06:04:27.115299+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/deploy.py', '--cfg-path', '/tmp/tmpgjvntqgv']
[2023-06-15T06:09:35.409+0000] {standard_task_runner.py:85} INFO - Job 36: Subtask pyspark_train_classifier_model
[2023-06-15T06:09:35.452+0000] {task_command.py:410} INFO - Running <TaskInstance: deploy_prediction_model.pyspark_train_classifier_model manual__2023-06-15T06:04:27.115299+00:00 [running]> on host 6426fae5ea9e
[2023-06-15T06:09:35.512+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deploy_prediction_model' AIRFLOW_CTX_TASK_ID='pyspark_train_classifier_model' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T06:04:27.115299+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T06:04:27.115299+00:00'
[2023-06-15T06:09:35.514+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-15T06:09:35.515+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\nspark-submit --class org.fiware.cosmos.orion.spark.connector.prediction.PredictionJob    --conf spark.driver.extraJavaOptions=-Dlog4jspark.root.logger=WARN,console   --master spark://spark-master:7077   //prediction-job/target/orion.spark.connector.prediction-1.0.1.jar   /']
[2023-06-15T06:09:35.524+0000] {subprocess.py:86} INFO - Output:
[2023-06-15T06:09:35.526+0000] {subprocess.py:93} INFO - /bin/bash: line 2: spark-submit: command not found
[2023-06-15T06:09:35.527+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2023-06-15T06:09:35.537+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2023-06-15T06:09:35.540+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=deploy_prediction_model, task_id=pyspark_train_classifier_model, execution_date=20230615T060427, start_date=20230615T060935, end_date=20230615T060935
[2023-06-15T06:09:35.551+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 36 for task pyspark_train_classifier_model (Bash command failed. The command returned a non-zero exit code 127.; 1470)
[2023-06-15T06:09:35.580+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-15T06:09:35.594+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
